### PR TITLE
add backup to acctd

### PR DESCRIPTION
**Pull request by v1peractual**
Add the ability to backup acctd node.

### CODE REVIEW

File: manifests/backup.pp [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# @summary Configure xCAT master backups
```
Comment (billglick) [inline manifests/backup.pp]:
```suggestion
# @summary Configure acctd backups
```

**Review by billglick — CHANGES_REQUESTED** [review]
Docstrings on new class refer to `xCAT master` instead of `acctd`.

File: manifests/backup.pp (hunk 1 above)
Comment (v1peractual) [reply manifests/backup.pp]: Fixed

File: REFERENCE.md [hunk 2]
```diff
@@ -7,6 +7,7 @@
 ### Classes
 
 * [`profile_acctd`](#profile_acctd): Manage resources related to NCSA acctd.
+* [`profile_acctd::backup`](#profile_acctd--backup): Configure xCAT master backups
```
Comment (billglick) [inline REFERENCE.md]: REFERENCE.md still needs to be run again...
`pdk bundle exec puppet strings generate --format markdown`

**Review by billglick — CHANGES_REQUESTED** [review]

File: REFERENCE.md (hunk 2 above)
Comment (v1peractual) [reply REFERENCE.md]:
```
kingda@Porsche:~/working/puppet-profile_acctd$ pdk bundle exec puppet strings generate --format markdown
pdk (INFO): Using Ruby 2.7.8
pdk (INFO): Using Puppet 7.25.0
Files                     2
Modules                   0 (0 undocumented)
Classes                   0 (0 undocumented)
Constants                 0 (0 undocumented)
Attributes                0 (0 undocumented)
Methods                   0 (0 undocumented)
Puppet Data Type Aliases  0 (0 undocumented)
Puppet Classes            2 (0 undocumented)
Puppet Types              0 (0 undocumented)
Puppet Data Types         0 (0 undocumented)
Puppet Functions          0 (0 undocumented)
Puppet Defined Types      0 (0 undocumented)
Puppet Providers          0 (0 undocumented)
Puppet Tasks              0 (0 undocumented)
Puppet Plans              0 (0 undocumented)
 100.00% documente
```

**Comment by v1peractual** [comment]
Updated the references.

**Review by billglick — APPROVED** [review]
This generally looks good, but I'm not real familiar with the acctd profile or functionality.

I defer to @jakerundall to comment if there are additional, default acctd `$locations` that might also be backed up.

**Comment by v1peractual** [comment]
I am not happy about just backing up root but that is what Glasgow said to do:

David King
  [4:15 PM](https://ncsa-at-illinois.slack.com/archives/D771GNNRJ/p1689887730992409)
Do you have a backup list for dt-alloc1 or acctd in general?

glasgow
  [4:16 PM](https://ncsa-at-illinois.slack.com/archives/D771GNNRJ/p1689887806954289)
Everything of any importance is in root's home directory now. If you are getting that we should be in good shape.
:+1:
1

**Pull request merged**